tests/components/pjlink/test_actions.py

```diff
@@ -0,0 +1,90 @@
+"""The test for the pjlink MediaEntity actions."""
```

Comment:
All the added tests belong in a module named `test_media_player.py`.

Reply:
Fixed